Packs/Sixgill-Darkfeed/Integrations/Sixgill_Darkfeed_Enrichment/Sixgill_Darkfeed_Enrichment.yml

```diff
@@ -0,0 +1,605 @@
+        - contextPath: SixgillDarkfeed.Actor.description
+          description: Description of IOC
+          type: String
+        - contextPath: SixgillDarkfeed.Actor.lang
+          description: Language of original post in Sixgill portal
+          type: String
+        - contextPath: SixgillDarkfeed.Actor.modified
+          description: Modification timestamp of IOC
+          type: Date
+        - contextPath: SixgillDarkfeed.Actor.pattern
+          description: IOC hash/domain/IP address. hashes include MD5, SHA-1 and SHA-256
+            when possible
+          type: String
+        - contextPath: SixgillDarkfeed.Actor.sixgill_actor
+          description: Actor of original post on dark web
+          type: String
+        - contextPath: SixgillDarkfeed.Actor.sixgill_confidence
+          description: Confidence score
+          type: Number
+        - contextPath: SixgillDarkfeed.Actor.sixgill_feedid
+          description: Subfeed ID
+          type: String
+        - contextPath: SixgillDarkfeed.Actor.sixgill_feedname
+          description: Subfeed name
+          type: String
+        - contextPath: SixgillDarkfeed.Actor.sixgill_postid
+          description: ID of post in Sixgill portal
+          type: String
+        - contextPath: SixgillDarkfeed.Actor.sixgill_posttitle
+          description: Title of post in Sixgill portal
+          type: String
+        - contextPath: SixgillDarkfeed.Actor.sixgill_severity
+          description: Severity score
+          type: Number
+        - contextPath: SixgillDarkfeed.Actor.sixgill_source
+          description: Source of post in Sixgill portal
+          type: String
+        - contextPath: SixgillDarkfeed.Actor.spec_version
+          description: STIX specification version
+          type: String
+        - contextPath: SixgillDarkfeed.Actor.type
+          description: STIX object type
+          type: String
+        - contextPath: SixgillDarkfeed.Actor.valid_from
+          description: Post creation date in Sixgill portal
+          type: Date
+        - contextPath: SixgillDarkfeed.Actor.labels
+          description: Indicative labels of IOC
+          type: Unknown
+        - contextPath: SixgillDarkfeed.Actor.external_reference
+          description: Link to IOC on Virustotal and abstraction of number of detections;
+            Mitre ATT&CK tatics and techniques
+          type: Unknown
+    - arguments:
+        - default: true
+          description: List of postid's
+          isArray: true
+          name: post_id
+          required: true
+          secret: false
+        - default: true
+          defaultValue: '0'
+          description: No. of indicators which need to be skipped while returning the
+            result set
+          isArray: false
+          name: skip
+          required: false
+          secret: false
+      deprecated: false
+      description: Return Postid information and reputation
+      execution: false
+      name: post_id
```

Comment:
`post_id` is not a valid reputation command, so it should follow conventions, i.e. vendorname-do-something instead of do-something.

For example `sixgill-get-post-id`